source/transactions-convenient-api/tests/commit-writeconcernerror.yml

```diff
@@ -228,34 +203,7 @@ tests:
         result:
           errorCodeName: UnsatisfiableWriteConcern
           errorLabelsOmit: ["TransientTransactionError", "UnknownTransactionCommitResult"]
-    expectations:
-      -
-        command_started_event:
-          command:
-            insert: *collection_name
-            documents:
-              - { _id: 1 }
-            ordered: true
-            lsid: session0
-            txnNumber: { $numberLong: "1" }
-            startTransaction: true
-            autocommit: false
-            # omitted fields
-            readConcern: ~
-            writeConcern: ~
-          command_name: insert
-          database_name: *database_name
-      -
-        command_started_event:
-          command:
-            commitTransaction: 1
-            lsid: session0
-            txnNumber: { $numberLong: "1" }
-            autocommit: false
-            # omitted fields
-            readConcern: ~
-            startTransaction: ~
-            writeConcern: ~
-          command_name: commitTransaction
-          database_name: admin
-    outcome: *outcome
+    expectations: *expectations_without_retries
```

Comment:
Reusing APM expectations here as well, since the sequence of commands for the UnknownReplWriteConcern and UnsatisfiableWriteConcern tests are identical (much like the first two tests in this file).